tools/test/audio/asrc_run.sh

```diff
@@ -2,13 +2,25 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright(c) 2020 Intel Corporation. All rights reserved.
 
-if [ -z "$6" ]; then
-    echo "Usage:   $0 <bits in> <bits out> <rate in> <rate out> <input> <output>"
-    echo "Example: $0 16 16 32000 48000 input.raw output.raw"
-    exit
-fi
+usage ()
+{
+    echo "Usage:   $1 <bits in> <bits out> <rate in> <rate out> <input> <output>"
+    echo "Example: $1 16 16 32000 48000 input.raw output.raw"
+}
 
-COMP=asrc
-DIRECTION=playback
+main()
+{
+    local COMP DIRECTION
 
-./comp_run.sh $COMP $DIRECTION $1 $2 $3 $4 $5 $6
+    if [ -z "$6" ]; then
```

Comment:
Nit:
```
if [ $# -gt 6 ]; then
```
So you can support a blank `$6` argument, which you probably don't want to so forget it :-)

Reply:
I guess you meant

`if [ $# -ne 6 ]; then
`
No problem, it's good to do once properly and learn good habits.

Reply:
> if [ $# -ne 6 ]; then

Yes, thanks for knowing better :-)

Reply:
Done